vpd-manager/src/manager_main.cpp

```diff
@@ -43,9 +46,40 @@ int main(int, char**)
 
         exit(EXIT_SUCCESS);
     }
-    catch (const std::exception& e)
+    catch (const std::exception& l_ex)
     {
-        std::cerr << e.what() << std::endl;
+        if (typeid(l_ex) == typeid(vpd::JsonException))
+        {
+            vpd::EventLogger::createAsyncPel(
```

Comment:
instead of calling vpd::EventLogger::createAsyncPel() API multiple places, 
Should we call at the end as we create PEL in all error scenario, 

If the input not going to change in the future, by just saving types::ErrorType in each if condition, what ever param changing ?

what's your thought on this ?

Reply:
We cannot call at the end, for some of the error type we need to pass procedure type as a parameter.

Reply:
Discussed in the scrum. Team is ok with the current implementation.